src/client/src/models/Employee.js

```diff
@@ -0,0 +1,11 @@
+export default class Employee {
+    constructor(id, name) {
+    this.id=id;
+    this.name=name;
+    }
+
+    Edit(id, name) {
```

Comment:
We're going to follow the convention of camalCase for variable and function names. Change "Edit" to "edit".